.github/workflows/artifacts-build.yml

```diff
@@ -141,7 +141,6 @@ jobs:
 
   wasm-demo:
     # This is too expensive to run on every push, so only run it on main.
-    # When running on a PR, comment this out and set the BASELINE variable below to the baseline commit.
```

Comment:
i assume you meant to comment out the next line?

Reply:
no